webapp/src/js/services/enketo.js

```diff
@@ -346,29 +346,40 @@ angular.module('inboxServices').service('Enketo',
       });
     };
 
-    const registerEditedListener = function(selector, listener) {
+    const registerEditedListener = function($selector, listener) {
       if (listener) {
-        $(selector).on('edited.enketo', listener);
+        $selector.on('edited.enketo', listener);
       }
     };
 
-    const registerValuechangeListener = function(selector, listener) {
+    const registerValuechangeListener = function($selector, listener) {
       if (listener) {
-        $(selector).on('valuechange.enketo', listener);
+        $selector.on('valuechange.enketo', listener);
       }
     };
 
+    const registerAddrepeatListener = function($selector, formDoc) {
+      $selector.on('addrepeat', function(ev) {
+        $timeout(() => { // timeout to allow enketo to finish first
+          replaceMediaLoaders($(ev.target), formDoc);
+        });
+      });
+    };
```

Comment:
The other alternative I considered was to block the Enketo init on the loading of the images. That way the image would be loaded once for the repeat template and then not loaded again. I think this way is more correct albeit slightly less performant.

Reply:
I agree this is more correct.